bin/zopen-clean

```diff
@@ -50,28 +52,32 @@ HELPDOC
 
 cleanUnused()
 {
-  needle=$1
-  [ -z "${needle}" ] && return 
-  current=$(getCurrentVersionDir "${needle}")
-
-  if [ -n "${current}" ]; then
-    current=$(basename "${current}")
-  else
-    printInfo "No currently active version of '${needle}'; removing all versions"
-  fi
-  cd "${ZOPEN_PKGINSTALL}/${needle}" && zosfind . -name "./*" -prune -type d |
-  while read repo; do
-    printVerbose "Parsing repo: '${repo}' as '${repo#./}'"
-    repo="${repo#./}"
-    if [ "${current}" = "${repo}" ]; then
-      printInfo "${NC}${GREEN}-> ${repo}  <- Current version${NC}"
+  for needle in $1; do
+    [ -z "${needle}" ] && return 
+    [ ! -e "${ZOPEN_PKGINSTALL}/${needle}" ] && printInfo "No versions of '${needle}' found" && continue
+    
+    current=$(getCurrentVersionDir "${needle}")
+  
+    if [ -n "${current}" ]; then
+      current=$(basename "${current}")
     else
-      deref=$(cd "${ZOPEN_PKGINSTALL}/${needle}/${repo}" && pwd -P)
-      [ "${deref}" = "/" ] || rm -rf "${deref}" > /dev/null 2>&1
-      printInfo "-- ${repo} <- Removed"
-      syslog "${ZOPEN_LOG_PATH}/audit.log" "${LOG_A}" "${CAT_FILE},${CAT_PACKAGE},${CAT_REMOVE}" "CLEAN" "cleanUnused" "Removed unused package at ${repo#"${ZOPEN_PKGINSTALL}"/} "
+      printInfo "No currently active version of '${needle}'; removing all versions"
     fi
-  done  
+
+    cd "${ZOPEN_PKGINSTALL}/${needle}" && zosfind . -name "./*" -prune -type d |
+    while read repo; do
+      printVerbose "Parsing repo: '${repo}' as '${repo#./}'"
+      repo="${repo#./}"
+      if [ "${current}" = "${repo}" ]; then
+        printInfo "${NC}${GREEN}-> ${repo}  <- Current version${NC}"
+      else
+        deref=$(cd "${ZOPEN_PKGINSTALL}/${needle}/${repo}" && pwd -P)
+        [ "${deref}" = "/" ] || rm -rf "${deref}" > /dev/null 2>&1
+        printInfo "-- ${repo} <- Removed"
```

Comment:
Should we add a prompt before removal?

Also, I wonder if any of the removal logic can be shared with zopen-remove and placed into a function in common.sh?